Source/Core/DolphinQt/Config/ConfigControls/ConfigSlider.h

```diff
@@ -38,6 +39,25 @@ class ConfigSlider final : public ConfigControl<ToolTipSlider>
   std::vector<int> m_tick_values;
 };
 
+class ConfigSliderU32 final : public ConfigControl<ToolTipSlider>
+{
+  Q_OBJECT
+public:
+  ConfigSliderU32(u32 minimum, u32 maximum, const Config::Info<u32>& setting, u32 scale = 1);
```

Comment:
As far as I can tell, the float slider uses `step` instead of `scale` but the value itself has the same meaning. Any reason for this inconsistency?

Reply:
FloatSlider is more complicated, and used for things like a 4x multiplier being displayed on the slider as a 400% increase. I see "step" as denoting changing how the slider works.

SliderU32 is just a multiplication before/after saving/getting, because the user-facing value and steps displayed on the slider are correct, but the stored value is scaled.  It allows for a simpler slider->value() check for the value label rather than needing extra math as well. So I was seeing "scale" as a simpler post-multi.

Only those two Sliders use it, so it's just written specifically for what they do.  If it still seems weird, let me know.